src/ventures-for-shared-prosperity/focus-area.njk

```diff
@@ -7,14 +7,138 @@ title: Focus area for ideas42 Venture Studio
 {% block masthead %}
   {{ super() }}
   <div class="hero standard-restricted-width">
-    <div class="g-max-3 g-right">
-      <p class="hero__tag">Where those who wish to focus come</p>
-      <h1 class="hero__title">
-      Focus Area
-      </h1>
+{%include "components/sub-banner.njk" %}
+        <p class="hero__tag">the poverty tax </p>
+        <h1 class="hero__title">
+          The Excess Costs of Poverty
+        </h1>
+        <p>
+          There are tens of millions of people in the U.S. who face chronic hunger, housing insecurity, inadequate healthcare, and a daily struggle to make ends meet. The causes of poverty are diverse, complex, and often structural, and the consequences are catastrophic at an individual and societal level.
+        </p>
+<div class="g-max-3 g-right">
+      <img src="" alt="Homeless encampment">
+      <img src="" alt="Swirly circle here">
     </div>
-  </div>
+    <div>
 
+      
+
+    <div>
```

Comment:
@will0101 Throughout this section there are a number of wrapping `div` elements? What semantic value are those adding? For this work, we want to focus on marking up the content, not adding html for structure or styling purposes.

Reply:
Fixed the _divfestation_.